podman-image-daily/rosetta-activation.sh

```diff
@@ -1,2 +1,11 @@
 #!/bin/bash
+set -e
+
+# mount rosetta tag, if the virtiofs tag does not exists do not return an error and exit early
+mount -t virtiofs -o context=system_u:object_r:nfs_t:s0 rosetta /mnt || exit 0
```

Comment:
If you enable rosetta unconditionally, could this go in a `mount` unit similar to the units used for virtiofs mounts? https://github.com/containers/podman/blob/9ac6d9de45ab18591a2bc38994ebda1d0d2bd2d0/pkg/machine/apple/apple.go#L86-L106

Reply:
Re-reading the PR description, you have different goals :) "Only use one unit to make the flow simpler"

Reply:
I tried using a mount unit first and while it worked the issue if rosetta is disabled in podman machine it would fail the mount unit and I find this ugly as this is really no error.